R/default_cleanepi_settings.R

```diff
@@ -41,7 +44,11 @@ get_default_params <- function() {
 modify_default_params <- function(defaults, params, strict = TRUE) {
   extra <- setdiff(names(params), names(defaults))
   if (strict && (length(extra) > 0L)) {
-    stop("Additional invalid options: ", toString(extra))
+    cli::cli_abort(c(
+      tr_("Found the following unrecognised arguments to {.fn clean_data}: {.emph {toString(extra)}}."), # nolint: line_length_linter
+      i = tr_("{.fn clean_data} does not support arguments other than the defaults. Run {.code get_default_params()} to display the list of default parameters."), # nolint: line_length_linter
```

Comment:
Consider splitting this line into 2 lines.

Reply:
I have accounted for this.